src/github.com/stellar/gateway/protocols/compliance/auth.go

```diff
@@ -38,8 +38,8 @@ type AuthData struct {
 	NeedInfo bool `json:"need_info"`
 	// The transaction that the sender would like to send in XDR format. This transaction is unsigned.
 	Tx string `json:"tx"`
-	// The full text of the memo the hash of this memo is included in the transaction.
-	Memo string `json:"memo"`
+	// The full text of the attachment. The hash of this attachment is included in the transaction memo.
+	Attach string `json:"attach"`
```

Comment:
I guess that's fine, but I do have an issue with it.  I don't get why we use `attach` in the actual json.  If we're concerned about the structures size, perhaps we shouldn't be using json.  As far as I can tell, this shouldn't effect the performance or scalability of the protocol in any way.

This all comes back to the same ideal that we should be communicating as clearly as we can.  Sometimes we have other constraints that get in the way, but I for one am not concerned about the 4 extra characters.